diff --git a/renderer/shaders/webgl/prefilterEnv.fs.glsl b/renderer/shaders/webgl/prefilterEnv.fs.glsl
index 25f5bdf..f766ed4 100644
--- a/renderer/shaders/webgl/prefilterEnv.fs.glsl
+++ b/renderer/shaders/webgl/prefilterEnv.fs.glsl
@@ -1,5 +1,6 @@
 #version 300 es
 precision mediump float;
+precision highp int;
 
 out vec4 FragColor;
 
@@ -70,4 +71,4 @@ void main() {
     prefilteredColor = prefilteredColor / totalWeight;
 
     FragColor = vec4(prefilteredColor, 1.0);
-}
\ No newline at end of file
+}
